lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -96,48 +96,50 @@
 '''
 
 EXAMPLES = r'''
-- name: List all networks associated to the YourOrg organization
-  meraki_network:
-    auth_key: abc12345
-    state: query
-    org_name: YourOrg
-  delegate_to: localhost
-- name: Query network named MyNet in the YourOrg organization
-  meraki_network:
-    auth_key: abc12345
-    state: query
-    org_name: YourOrg
-    net_name: MyNet
-  delegate_to: localhost
-- name: Create network named MyNet in the YourOrg organization
-  meraki_network:
-    auth_key: abc12345
-    state: present
-    org_name: YourOrg
-    net_name: MyNet
-    type: switch
-    timezone: America/Chicago
-    tags: production, chicago
-  delegate_to: localhost
-- name: Create combined network named MyNet in the YourOrg organization
-  meraki_network:
-    auth_key: abc12345
-    state: present
-    org_name: YourOrg
-    net_name: MyNet
-    type:
-      - switch
-      - appliance
-    timezone: America/Chicago
-    tags: production, chicago
-- name: Enable VLANs on a network
-  meraki_network:
-    auth_key: abc12345
-    state: query
-    org_name: YourOrg
-    net_name: MyNet
-    enable_vlans: yes
-  delegate_to: localhost
+- delegate_to: localhost
+  block:
+    - name: List all networks associated to the YourOrg organization
+      meraki_network:
+        auth_key: abc12345
+        state: query
+        org_name: YourOrg
+      delegate_to: localhost
```

Comment:
`delegate_to` at target level should be removed (tasks will inherit `deletage_to` from parent block).